phpBB/styles/prosilver/template/posting_body.html

```diff
@@ -1,6 +1,11 @@
-
 <!-- IF S_PRIVMSGS -->
 	<!-- INCLUDE posting_pm_layout.html -->
 <!-- ELSE -->
+	<!-- INCLUDEJS {T_ASSETS_PATH}/javascript/jquery.caret.min.js -->
```

Comment:
No, you don't need to with INCLUDEJS; https://github.com/phpbb/phpbb/blob/c571d99c0f4108ec9136b601c9a57b4878597470/phpBB/phpbb/template/twig/node/includeasset.php#L53